From f8d788e35c30713118ceb87a32770d35ab794249 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Wed, 22 Nov 2017 20:47:01 +0000
Subject: [PATCH 1/1] Add StartupWMClass to desktop file

* Fixes the ability of pinning to a panel and the icon to double appear
  in a dock.

Signed-off-by: Joey Riches <josephriches@gmail.com>
---
 data/me.mitya57.ReText.desktop | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/me.mitya57.ReText.desktop b/data/me.mitya57.ReText.desktop
index e5e0794..f761aec 100644
--- a/data/me.mitya57.ReText.desktop
+++ b/data/me.mitya57.ReText.desktop
@@ -28,5 +28,6 @@ Categories=Office;WordProcessor;
 Exec=retext %F
 Type=Application
 Icon=retext
+StartupWMClass=ReText
 MimeType=text/markdown;text/x-rst;
 Keywords=Text;Editor;Markdown;reStructuredText;
-- 
2.15.0

